lib/kitto/router.ex

```diff
@@ -78,7 +78,7 @@ defmodule Kitto.Router do
   post "widgets/:id", private: %{authenticated: true} do
     {:ok, body, conn} = read_body(conn)
 
-    Notifier.broadcast!(id, body |> Poison.decode!)
+    Notifier.broadcast!(id |>  String.to_atom, body |> Poison.decode!(keys: :atoms!))
```

Comment:
This creates a DOS hole as Elixir only has a limited atom table (default is 1,048,576) and atoms are not garbage collected.